### DISCUSSION TITLE
How to make a custom compare to validation rule for dates?

### DISCUSSION THREAD

**Discussion by deanCompRSA** [vueform/vueform#283] [Questions] [answered]
I need to make a custom validation that will check if the date of one field, is lesser, greater or equal to the date value of another field.

Only one example in the docs are shown for a uppercase test where the value is checked. But, how do I check the value against another's value?

**Comment by adamberecz** [comment, marked as answer]
We have the following rules available for this:
- https://vueform.com/docs/validating-elements#rule-after
- https://vueform.com/docs/validating-elements#rule-after_or_equal
- https://vueform.com/docs/validating-elements#rule-before
- https://vueform.com/docs/validating-elements#rule-before_or_equal

If you still need a custom one, you can check out in the source how they're implemented:
https://github.com/vueform/vueform/blob/main/src/services/validation/rules/after.js
